Libraries/AppState/AppState.js

```diff
@@ -26,9 +26,6 @@ const invariant = require('fbjs/lib/invariant');
  * AppState is frequently used to determine the intent and proper behavior when
  * handling push notifications.
  *
- * This module depends on the native RCTAppState module. If you don't include it,
```

Comment:
Removing this from here since I don't want this to appear in open source documentation.